packaging/debian/rules

```diff
@@ -53,7 +53,7 @@ override_dh_auto_install:
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbmctl $(TMP)/pbmctl
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbm-agent $(TMP)/pbm-agent
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbm-coordinator $(TMP)/pbm-coordinator
-	cp -f packaging/conf/pbm-agent-storages.conf  $(TMP)/pbm-agent-storages.conf
+	cp -f packaging/conf/pbm-agent-storage.conf  $(TMP)/pbm-agent-storages.conf
```

Comment:
this:
$(TMP)/pbm-agent-storages.conf
needs to be:
$(TMP)/pbm-agent-storage.conf

Reply:
fixed